dashboard/src/features/testnet/reducers.js

```diff
@@ -20,9 +20,9 @@ export const blockchainId = (state = 0, action) => {
   return state
 }
 
-export const rpcVersion = (state = 0, action) => {
+export const crosscoreRpcVersion = (state = 0, action) => {
   if (action.type == 'TESTNET_CONFIG') {
-    return action.data.network_rpc_version
+    return action.data.crosscore_rpc_version || action.data.network_rpc_version
```

Comment:
Did a double-take that this wasn't camelCased, but it's actually an out-of-band fetch request.